dist/spec.json

```diff
@@ -13,7 +13,7 @@
       "name": "Apache-2.0",
       "url": "https://www.apache.org/licenses/LICENSE-2.0.html"
     },
-    "version": "2.10.0"
+    "version": "2.11.0"
```

Comment:
The version will be 2.12.0 as I released 2.11.0 for the IDP policy rules models.

Reply:
Never mind, this will likely be in 2.11.0